server/tests/participant.form.test.js

```diff
@@ -1,674 +1,72 @@
-/* eslint-disable no-restricted-syntax, no-await-in-loop */
-const { readFileSync } = require('fs');
-const { join } = require('path');
-const { ValidationError } = require('yup');
-const { v4 } = require('uuid');
+const request = require('supertest');
```

Comment:
Man I split the commits in the hopes that GH would handle this, but it didn't. This file is new so please review as such 🙏